.github/workflows/backport.yml

```diff
@@ -22,7 +22,8 @@ jobs:
           installation_id: 22958780
 
       - name: Backport
-        uses: VachaShah/backport@v1.1.4
+        uses: VachaShah/backport@v2.1.0
         with:
           github_token: ${{ steps.github_app_token.outputs.token }}
           branch_name: backport/backport-${{ github.event.number }}
```

Comment:
This should be `head_template: backport/backport-<%= number %>-to-<%= base %>` instead of branch_name.

Reply:
Updated.